doc/configuring.md

```diff
@@ -0,0 +1,361 @@
+* pre-processes and trims it;
+* runs `sourmash gather` on the metagenome, using one or more sourmash databases, to find a [minimum metagenome cover](https://www.biorxiv.org/content/10.1101/2022.01.11.475838v1);
+* retrieves the full genomes for any matches and executes a variety of mapping-based analyses;
+* incorporates taxonomy information for the genomes into the taxonomy summary report (if taxonomy reporting is requested).
+
+Much of the configuration for genome-grist is about where to find more information about matching genomes.
+
+For Genbank genomes, this is easy! But if you're providing your own genomes and taxonomy information, it's a bit trickier.
+
+## Using Genbank genomes
+
+For Genbank genomes, all the necessary information is available already, or automatically determined by genome-grist!
+
+sourmash already provides pre-built databases containing [all GTDB genomes (R06 rs202)](https://sourmash.readthedocs.io/en/latest/databases.html) as well as [all 700,000 Genbank microbial genomes from July 2020](https://github.com/sourmash-bio/sourmash/issues/1749#issuecomment-947920226).
+
+For genomes available through Genbank (aka with Genbank accessions), genome-grist does the genome retrieval automatically, so you don't need to have them downloaded already.
+
+Taxonomy spreadsheets are available for GTDB (at the databases page) and for Genbank 700k/July 2020 (link upon request).
+
+## Preparing information on private genomes
+
+If you want to use unpublished or private genomes with genome-grist, you'll need to provide your own sourmash database, your own set of genome files and information, and your own taxonomy spreadsheet. Luckily this is all pretty straightforward and we provide tools to help you! Read on!
+
+Note that you can absolutely combine Genbank with your own databases here, or just use your own databases. (If there are overlapping identifiers, the private genomes are chosen first; you might want to do this if you already have a bunch of the Genbank genomes downloaded already, for example.)
+
+### Choosing identifiers for your genomes
+
+You'll need to choose unique identifiers for your genomes. genome-grist requires that your identifier does not have a space, colon (:), or forward slash (/) in it; everything else should be fine.
+
+### Preparing your genome files
+
+You'll need one FASTA file per genome (gzip or bz2 compressed is fine). The filename doesn't matter. It's probably easiest if they're all in one directory, although this isn't necessary.
+
+We suggest naming at the first sequence in each FASTA file with the identifier at the start, space delimited - for example `MY_ID_1.1 first_sequence_name is very special` .
+
+### Creating one or more sourmash databases
+
+You'll need to provide at least one sourmash database for your private collection to genome-grist under the config parameter `private_databases`, which takes a list of paths to sourmash database locations.
```

Comment:
Can you provide multiple databases, one being your private collection and one being e.g. GTDB?

Reply:
yep. will work on making this clearer.

Actually, it makes me wonder if we even need to distinguish between private and genbank databases at this point in the config. Will revisit.

Reply:
On farm for example, I can use the precomputed genbank as private genomes. So, I think *local genomes* will be more descriptive here.

Reply:
added to checklist to consider, and updated text to read:
>You can mix private databases with genbank databases, but 
you'll need to provide one or more sourmash databases for any private collections. You can do this with the config parameter `private_databases`, which takes a list of paths to sourmash database locations.

Reply:
ahh, missed @mr-eyes comment which hit between me typing mine and me hitting 'resolve' :).

I consider this one of those fine distinctions that experts will be able to figure out immediately, but will only confuse newbies. See also the text above,
Note that you can absolutely combine Genbank with your own databases here, or just use your own databases. (If there are overlapping identifiers, the private genomes are chosen first; you might want to do this if you already have a bunch of the Genbank genomes downloaded already, for example.)

I do recognize that there is potential confusion between local, private, unpublished, etc. and I've tried to write it somewhat clearly, but I'm done for now :)